.test-infra/jenkins/job_InferenceBenchmarkTests_Python.groovy

```diff
@@ -0,0 +1,170 @@
+        bert_tokenizer        : 'bert-base-uncased',
+        model_state_dict_path : 'gs://apache-beam-ml/models/huggingface.BertForMaskedLM.bert-base-uncased.pth',
+        output                : 'gs://temp-storage-for-end-to-end-tests/torch/result_' + now + '.txt',
+      ]
+    ],
+    [
+      title             : 'Pytorch Langauge Modeling using Hugging Face bert-large-uncased model',
+      test              : 'apache_beam.testing.benchmarks.inference.pytorch_language_modeling_benchmarks',
+      runner            : CommonTestProperties.Runner.DATAFLOW,
+      pipelineOptions: [
+        job_name              : 'benchmark-tests-pytorch-language-modeling-bert-large-cased' + now,
+        project               : 'apache-beam-testing',
+        region                : 'us-central1',
+        staging_location      : 'gs://temp-storage-for-perf-tests/loadtests',
+        temp_location         : 'gs://temp-storage-for-perf-tests/loadtests',
+        requirements_file     : 'apache_beam/ml/inference/torch_tests_requirements.txt',
+        pickle_library       : 'cloudpickle',
+        experiments           : 'no_use_multiple_sdk_containers',
+        publish_to_big_query  : true,
+        metrics_dataset       : 'beam_run_inference',
+        metrics_table         : 'torch_language_modeling_bert_large_uncased',
+        input_options         : '{}', // this option is not required for RunInference tests.
+        influx_measurement    : 'torch_language_modeling_bert_large_uncased',
+        influx_db_name        : InfluxDBCredentialsHelper.InfluxDBDatabaseName,
+        influx_hostname       : InfluxDBCredentialsHelper.InfluxDBHostUrl,
+        // args defined in the example.
+        input                 : 'gs://apache-beam-ml/testing/inputs/sentences_50k.txt',
+        // TODO: make sure the model_state_dict_path weights are accurate.
+        bert_tokenizer        : 'bert-large-uncased',
+        model_state_dict_path : 'gs://apache-beam-ml/models/huggingface.BertForMaskedLM.bert-large-uncased.pth',
+        output                : 'gs://temp-storage-for-end-to-end-tests/torch/result_' + now + '.txt'
+      ]
+    ],
+  ]
+}
+
+def loadTestJob = { scope ->
+  List<Map> testScenarios = loadTestConfigurations()
+  for (Map testConfig: testScenarios){
+    commonJobProperties.setTopLevelMainJobProperties(scope, 'master', 180)
+    loadTestsBuilder.loadTest(scope, testConfig.title, testConfig.runner, CommonTestProperties.SDK.PYTHON, testConfig.pipelineOptions, testConfig.test, null, testConfig.pipelineOptions.requirements_file)
+  }
+}
+
+PhraseTriggeringPostCommitBuilder.postCommitJob(
+    'beam_Inference_Python_Benchmarks_Dataflow',
+    'Run Inference Benchmarks',
+    'Inference benchmarks on Dataflow(\"Run Inference Benchmarks"\"")',
+    this
+    ) {
+      loadTestJob(delegate)
+    }
+
+// TODO(anandinguva): Change the cron job to run once a day
+CronJobBuilder.cronJob(
+    'beam_Inference_Python_Benchmarks_Dataflow', 'H 2 * * *',
```

Comment:
what is `H 2 * *`

Reply:
Way of specifying cron jobs that runs every two hours in this case. I will change this to 24 hours.